### PR TITLE
add cloudfront distribution

### PR DESCRIPTION
**Pull request by jharewinton**
cloudfront cloudformation is in a separate file as it has to be a stack in N.Viriginia

### CODE REVIEW

File: cloudformation/cloudfront-cf.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      DistributionConfig:
+        Origins:
+        - DomainName: !Ref 'OriginBucketSite'
+          Id: help
+          S3OriginConfig:
+            OriginAccessIdentity: !Join ['', ['origin-access-identity/cloudfront/', !Ref 'OriginAccessId']]
+        Enabled: 'true'
+        DefaultCacheBehavior:
+          AllowedMethods:
+          - DELETE
```
Comment (gtrufitt) [inline cloudformation/cloudfront-cf.yaml]: Not sure if this matters, but does it need all these methods?

**Review by gtrufitt — COMMENTED** [review]
This looks ok to me but maybe someone better at cloudformation stuff could have a look?

File: cloudformation/cloudfront-cf.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+Resources:
+  distribution:
+    Type: AWS::CloudFront::Distribution
+    Properties:
+      DistributionConfig:
+        Origins:
+        - DomainName: !Ref 'OriginBucketSite'
+          Id: help
+          S3OriginConfig:
+            OriginAccessIdentity: !Join ['', ['origin-access-identity/cloudfront/', !Ref 'OriginAccessId']]
```
Comment (mbarton) [inline cloudformation/cloudfront-cf.yaml]: You can use `!Sub` to make interpolation a bit more readable:

```yaml
OriginAccessIdentity: !Sub 'origin-access-identity/cloudfront/${OriginAccessId}'
```

File: cloudformation/cloudfront-cf.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    Description: ACM certificate for CloudFront
+    Type: String
+  S3BucketName:
+    Description: Name of the S3 Bucket
+    Type: String
+    Default: 'editorial-tools-help'
+  OriginAccessId:
+    Description: Id of OriginAccessIdentity user
+    Type: String
+    Default: 'EOQ223G6SK3D5'
```
Comment (mbarton) [inline cloudformation/cloudfront-cf.yaml]: Should this default be in a public repo?

**Review by akash1810 — COMMENTED** [review]
can the cfn go here - https://github.com/guardian/editorial-tools-platform/tree/master/cloudformation

**Comment by jharewinton** [comment]
@akash1810 good point. PR for the change here: https://github.com/guardian/editorial-tools-platform/pull/91

**Pull request merged**